widget/app.services.js

```diff
@@ -75,7 +75,45 @@
             }
           });
           return deferred.promise;
-        }, onUpdate: function (viewName) {
+        },getAll: function (options, _tagName) {
```

Comment:
There is no lazy loading right? We are fetching everything at once? Is there specific reason why we fetch all at once?

Reply:
When user is on Map view and they want to see all the locations, we are fetching everything.